content/developer-platform/service-providers/publish-resources/_index.en.md

```diff
@@ -401,10 +402,10 @@ Note that fields like `generation` or `resourceVersion` are not relevant for any
 The sync loop can be divided into 5 parts:
 
 1. find the local object
-2. handle deletion
-3. ensure the destination object exists
-4. ensure the destination object's content matches the source object
-5. synchronize related resources the same way (repeat 1-4 for each related resource)
+1. handle deletion
+1. ensure the destination object exists
+1. ensure the destination object's content matches the source object
+1. synchronize related resources the same way (repeat 1-4 for each related resource)
```

Comment:
is it some kind of automated numbered list destroyed you've run? 😂

Reply:
PTAL at the netlify preview. https://deploy-preview-1964--cranky-newton-4e6ed2.netlify.app/developer-platform/service-providers/publish-resources/#reconcile-loop. Ordering is not broken here,  it is fix for remark-lint-ordered-list-marker-style warnings.  Thanks! :)